docs/sdks/python/agents.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: AI Agents with LLMs and Skills
+sidebarTitle: AI Agents
+---
+
+With MindsDB, you can create and deploy AI agents that comprise AI models and customizable skills such as knowledge bases and text-to-SQL.
+
+<p align="center">
+  <img src="/assets/agent_diagram.png" />
+</p>
+
+AI agents use a conversational model (like OpenAI) from LangChain utilizing [tools as skills](https://python.langchain.com/docs/modules/agents/tools/) to respond to user input. Users can customize AI agents with their own prompts to fit their use cases.
+
+## AI Agents' Components
+
+AI agents comprise of skills, such as text2sql and knowledge_base, and a conversational model.
+
+### Skills
+
+Skills provide data resources to an agent, enabling it to answer questions about available data.
+
+Here is how to create and work with skills:
+
+```bash
+# get a skill by name
+skill = skills.get('my_skill')
+
+# list all skills
+skills = skills.list()
+
+# create a new SQL skill
+text_to_sql_skill = skills.create('text_to_sql', 'sql', { 'tables': ['my_table'], 'database': 'my_database' })
+
+# update a skill
+skill.params = { 'tables': ['new_table'], 'database': 'new_database' }
+updated_skill = skills.update('my_skill', skill)
+
+# delete a skill by name
+skills.delete('my_skill')
+```
+
+#### Text2SQL
+
+To search data efficiently, use the text2sql skills that translate user's input into a query and fetch relevant data from defined datasets.
```

Comment:
It looks like this section is formatted as code. Or everything works through mintlify? Also maybe we can change the ```bash to python